src/client/components/DiceTemplates.tsx

```diff
@@ -373,6 +392,20 @@ function DicePicker() {
   const [diceHolder, setDiceHolder] = useState<RRDiceTemplatePart[]>([]);
   const diceParts = [4, 6, 8, 10, 12, 20].map((faces) => makeDicePart(faces));
 
+  const proficiencyPart: RRDiceTemplatePartLinkedProficiency = {
+    id: rrid<RRDiceTemplatePart>(),
+    type: "linkedProficiency" as const,
+    damage: { type: null },
+    proficiency: 1,
+  };
+
+  const initiativePart: RRDiceTemplatePartLinkedModifier = {
+    id: rrid<RRDiceTemplatePart>(),
+    type: "linkedModifier" as const,
+    damage: { type: null },
+    name: "initiative",
+  };
```

Comment:
Should we make these two functions so that we do not re-use template parts with the same id?

Reply:
Okay, I made them into functions now, but had to assign them as we need a reference later. Is that fine?

Reply:
Oh, I'm sorry, After looking at the code again, it seems like it was just fine the way you had it initially 🙈 since we overwrite these ids before adding parts to a proper template:
https://github.com/cmfcmf/roc-and-roll/blob/ced4c7ca91ea7659ef0aad0cc4180512c181a436/src/client/components/DiceTemplates.tsx#L99
https://github.com/cmfcmf/roc-and-roll/blob/ced4c7ca91ea7659ef0aad0cc4180512c181a436/src/client/components/DiceTemplates.tsx#L568-L593

I have some thoughts on how to make this less likely to break in the future, but that's unrelated to this PR. I think we can just leave it as it is for now.

Reply:
Alright, will undo it for now then